packages/localization/src/languages/en.json

```diff
@@ -496,6 +496,8 @@
                 "uploadSignatoryId": "Upload Singatory ID",
                 "titleConnectOrRegisterIREC": "Thank you for registering organization on TK marketplace!",
                 "contentConnectOrRegisterIREC": "We are checking your information as soon as possible and will contact you once everything is approved and you can start trading.<1/> <2/> In order to register devices and request I-RECs, users also need to connect to existing I-REC account.",
+                "titleIRecInfoSuccess": "Thanks for providing your I-REC account information.",
```

Comment:
Formal `Thank you` instead of informal `Thanks`.

Reply:
> @alexworker23 Could you please attach screenshots to you PRs when applicable? Would help a lot with reviewing.
> Thanks.

Ok, will do it from now on. "Thanks" - changed.